demisto_sdk/commands/generate_docs/tests/generate_docs_test.py

```diff
@@ -193,6 +193,20 @@ def test_get_used_in():
 # integration tests
 
 
+def test_generate_integration_general_section():
```

Comment:
this doesn't test the function (you are not even calling it...)

Reply:
because there is not a separate function for the general part.